pkg/outputs/outputs.go

```diff
@@ -6,6 +6,7 @@ var (
 	// Airflow is the output type for airflow readings.
 	Airflow = output.Output{
 		Name:      "airflow",
+		Type:      "speed",
```

Comment:
not sure if "speed" is necessarily correct, but its not technically a velocity since there is no directionality (at least not explicitly... +/- could maybe be considered directionality), and its not really a volumetric flow since its just mm/s (linear), not mm^3/second (cubic) or something like that.